src/panels/lovelace/common/handle-action.ts

```diff
@@ -41,7 +41,11 @@ export const handleAction = async (
   }
 
   const actionConfigs =
-    actionConfig.action === "multiple" ? actionConfig.actions : [actionConfig];
+    actionConfig.action === "multiple"
+      ? Array.isArray(actionConfig.actions)
```

Comment:
<https://github.com/home-assistant/frontend/blob/dev/src/common/ensure-array.ts>

Reply:
How I googled for this, which I assumed must be in js/ts somewhere. Should have git greped...